clients/client-java/src/test/java/com/datastrato/gravitino/client/TestGravitinoVersion.java

```diff
@@ -70,4 +71,22 @@ void testVersionCompare() {
     version2 = new GravitinoVersion("2.5.3-SNAPSHOT", "2023-01-01", "1234567");
     assertEquals(0, version1.compareTo(version2));
   }
+  @Test
+    void testVersionEquals() {
+      //test equal
+      GravitinoVersion version1 = new GravitinoVersion("3.1.3", "2022-01-01", "1234567");
+      GravitinoVersion version2 = new GravitinoVersion("3.1.3", "2022-01-01", "1234567");
+      assertTrue(version1.equals(version2));
+  
+      //test not equal
+      version1 = new GravitinoVersion("3.1.4", "2022-01-01", "1234567");
+      version2 = new GravitinoVersion("3.1.2", "2022-01-01", "1234567");
+      assertFalse(version1.equals(version2));
+  
+      //test equal with suffix
+      version1 = new GravitinoVersion("3.1.6-BETA", "2022-01-01", "1234567");
+      version2 = new GravitinoVersion("3.1.6", "2022-01-01", "1234567");
```

Comment:
I don't think the versions "3.1.6" and "3.1.6-BETA" are the same. The way you do comparison using parsed version number integer array seems not correct.